qcommon/src/org/qcmg/common/model/ProfileType.java

```diff
@@ -101,7 +103,7 @@ public static ProfileType getType2(String s) {
 	    	ext = s.substring(i+1).toLowerCase();
 	    }
 	    
-	    if (BAM_EXTENSION.equals(ext) || SAM_EXTENSION.equals(ext))
+	    if (BAM_EXTENSION.equals(ext) || SAM_EXTENSION.equals(ext) || ext.contentEquals(CRAM_EXTENSION))
```

Comment:
why contentEquals and not equals like the others?

Reply:
typo fixed